source/sass/blocks/slider.scss

```
@@ -37,85 +40,128 @@
 .slider__controls {
   position: relative;
 
-  margin-left: 20px;
-  margin-right: 17px;
+  display: inline-flex;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.slider__scale {
+  --scale-width: 84px;
+  --scale--margins: 13px;
+  --border-sum-width: 2px;
+
+  position: relative;
+
+  width: var(--scale-width);
+  height: 14px;
 
-  width: 82px;
-  height: 12px;
+  margin: 0 var(--scale--margins);
 
   border: $border-count;
-  background-color: $basic-white;
   border-radius: 6px;
 
+  background-color: $basic-white;
+
   @media (min-width: $tablet-width) {
-    margin-top: 18px;
+    --scale-width: 428px;
+    --scale--margins: 24px;
 
-    width: 428px;
     height: 6px;
-
     background-color: $slider-color;
-    border: 0;
+
+    border: none;
   }
 }
 
-.slider__bar {
+.slider__toggle {
+  --toggle-width: 35px;
+  --left-position: 3px;
+  --right-position: calc(
+    var(--scale-width) - var(--left-position) - var(--toggle-width) -
+      var(--border-sum-width)
+  );
+
+  --range-position: --left-position;
```

```suggestion
  --range-position: var(--left-position);
```
Применяются переменные обязательно вызовом функции var